flood_adapt/object_model/hazard/forcing/wind.py

```diff
@@ -138,3 +140,29 @@ class WindMeteo(IWind):
     @classmethod
     def default(cls) -> "WindMeteo":
         return WindMeteo()
+
+
+class WindNetCDF(IWind):
+    source: ForcingSource = ForcingSource.NETCDF
+    unit: us.UnitTypesVelocity = us.UnitTypesVelocity.mps
+
+    path: Path
+
+    def read(self) -> xr.Dataset:
+        ds = xr.open_dataset(self.path)
+        required_vars = {"wind10_v", "wind10_u", "press_msl"}
+        required_coords = {"time", "lat", "lon"}
+        return validate_netcdf_forcing(ds, required_vars, required_coords)
+
+    def save_additional(self, output_dir: Path | str | os.PathLike) -> None:
+        if self.path:
+            output_dir = Path(output_dir)
+            if self.path == output_dir / self.path.name:
```

Comment:
you might want to normalize the path here so that /foo/bar/ and /foo/bar/../bar will end up as the same